<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">/* === Background for the main dialog (modern hospital style) === */
QDialog {
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:0, y2:1,
        stop:0 #dff1ff,   /* light hospital blue */
        stop:1 #ffffff    /* clean white */
    );
    border: 1px solid #c7d9eb;
    border-radius: 12px;
    padding: 15px;
}

/* === Header label === */
QLabel#labelheader {
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 26px;
    font-weight: 600;
    color: #1a73e8; /* hospital blue */
    background-color: transparent;
    qproperty-alignment: 'AlignCenter';
}

/* === Username and Password labels === */
QLabel#labeluser,
QLabel#labelpass {
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 15px;
    color: #2c3e50; /* dark grey for contrast */
    margin-bottom: 4px;
}

/* === Username and Password inputs === */
QLineEdit#txtUsername,
QLineEdit#txtPassword {
    background-color: #fefefe;
    border: 1px solid #cbd6e2;
    border-radius: 6px;
    padding: 6px 10px;
    font-size: 15px;
    min-height: 28px;
    color: #2c3e50;
}

QLineEdit#txtUsername:focus,
QLineEdit#txtPassword:focus {
    border: 1px solid #1a73e8;
    background-color: #ffffff;
    outline: none;
}

/* === Login button (modern gradient button) === */
QPushButton#btnLogin {
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:0, y2:1,
        stop:0 #4facfe,    /* modern light blue top */
        stop:1 #00f2fe     /* clean cyan bottom */
    );
    color: #ffffff;
    border-radius: 6px;
    padding: 8px 20px;
    font-size: 15px;
    font-weight: 600;
    border: none;
    transition: background-color 0.3s ease;
}

QPushButton#btnLogin:hover {
    background-color: qlineargradient(
        spread:pad, x1:0, y1:0, x2:0, y2:1,
        stop:0 #00c6fb,
        stop:1 #005bea
    );
}

QPushButton#btnLogin:pressed {
    background-color: #004e92;
}
</string>
     </property>
     <widget class="QLabel" name="labeluser">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelpass">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelheader">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>481</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Hospital Mangement Service</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtUsername">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>281</width>
        <height>42</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>230</y>
        <width>281</width>
        <height>42</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
